gradle-plugin/it/src/it/micronaut-properties/build.gradle

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+plugins {
+    id("com.github.johnrengelman.shadow") version "8.1.1"
+    id("io.micronaut.application") version "4.1.0"
+    id("org.eclipse.jkube.kubernetes") version "1.17.0"
+    id("org.eclipse.jkube.openshift") version "1.17.0"
+}
+
+version = "0.1"
+group = "org.eclipse.jkube.integration.tests.gradle.micronaut"
+
+repositories {
+    mavenCentral()
+}
+
+micronaut {
+    runtime("netty")
+    processing {
+        incremental(true)
+        annotations("org.eclipse.jkube.integration.tests.gradle.micronaut.*")
+    }
+}
+
+dependencies {
+    annotationProcessor("io.micronaut:micronaut-http-validation")
+    annotationProcessor("io.micronaut.validation:micronaut-validation-processor")
+    annotationProcessor("io.micronaut:micronaut-inject-java")
+    implementation("io.micronaut:micronaut-http-client")
+    implementation("io.micronaut:micronaut-management")
+    implementation("io.micronaut:micronaut-runtime")
+    implementation("jakarta.annotation:jakarta.annotation-api")
+    implementation("io.micronaut.validation:micronaut-validation")
+    implementation("io.micronaut:micronaut-jackson-databind")
+    runtimeOnly("ch.qos.logback:logback-classic")
+
+}
+
+
+application {
+    mainClass.set("org.eclipse.jkube.integration.tests.gradle.micronaut.Application")
+}
+java {
+    sourceCompatibility = JavaVersion.toVersion("17")
+    targetCompatibility = JavaVersion.toVersion("17")
```

Comment:
I think you've added this because of [Micronaut 4 JDK 17](https://micronaut.io/2023/02/16/micronaut-framework-4-0-with-java-17-baseline/) baseline requirement. This will fail on JDK 11 . Not sure whether we should move this test to  micronaut 3

Reply:
Should we roll back to Micronaut 3 for the time being? Is there any hard limit on the JDK version for IT?

Reply:
I'll check with Marc about this and get back to you.

Reply:
Let's keep this Java11 friendly so we can merge it.

Reply:
@arman-yekkehkhani : Could you please use micronaut 3 instead?

Reply:
@rohanKanojia @manusa Thanks! changed Micronaut 4 to 3.